plugins/destination/duckdb/client/write.go

```diff
@@ -164,15 +152,61 @@ func (c *Client) WriteTableBatch(ctx context.Context, table *schema.Table, recor
 	return c.upsert(ctx, tmpTableName, table)
 }
 
+func writeTMPFile(table *schema.Table, records []arrow.Record) (fileName string, err error) {
+	sc := transformSchemaForWriting(table.ToArrowSchema())
+
+	// create temp file
+	f, err := os.CreateTemp("", fmt.Sprintf("%s-*.parquet", table.Name))
+	if err != nil {
+		return "", err
+	}
+	defer f.Close() // we don't care here, as the happy-path will actually check the error
+	fileName = f.Name()
+
+	// prep file writer
+	fw, err := pqarrow.NewFileWriter(sc, f,
+		parquet.NewWriterProperties(
+			parquet.WithVersion(parquet.V2_4),
```

Comment:
why not `V2_LATEST` (which is `V2_6`?)

Reply:
8e634aca3bce98b3c0c7b3c8135b38d9a5710669